modules/apps/questions/questions-web/test/js/utils/utils.es.js

```diff
@@ -21,5 +21,34 @@ describe('utils', () => {
 				'lorem ipsum dolor'
 			);
 		});
+
+		it('return a text with a hyphen and a space', () => {
+			expect(Utils.slugToText('Lorem--ipsum-dolor')).toEqual(
+				'lorem-ipsum dolor'
+			);
+			expect(Utils.slugToText('Lorem--ipsum--dolor')).toEqual(
+				'lorem-ipsum-dolor'
+			);
+		});
+	});
+
+	describe('stringToSlug', () => {
+		it('return a text with hyphens instead of spaces', () => {
+			expect(Utils.stringToSlug('Lorem ipsum dolor')).toEqual(
+				'lorem-ipsum-dolor'
+			);
+			expect(Utils.stringToSlug('Lorem  ipsum  dolor')).toEqual(
```

Comment:
We add the trim functionality?

Reply:
Nop. Right now, what we do is replace the white spaces ('\s') for a hyphen, it doesn't matter how many white spaces contain the text. So, it is not totally a trim.